src/http/oauth1.ts

```diff
@@ -204,6 +204,8 @@ export function createOAuth1HttpClient(
    */
   return Object.defineProperty(
     {
+      skipCaseNormalizationForCustomProperties:
+        options.skipCaseNormalizationForCustomProperties !== false,
```

Comment:
Check is like this because of the negative in the name. `undefined` should be true.

Reply:
Isn't it kind of weird that we have `skipKeyNormalizationForProperties` and `skipCaseNormalizationForCustomProperties`? 

Aren't they the same thing, but targeting different things? or do the names key vs case mean something different?

Reply:
`skipCaseNormalizationForCustomProperties` is a setting we expose where we try to explain to the customer about which type of properties this is about. The custom ones. That is just a boolean they can turn off and on again.

`skipKeyNormalizationForProperties` is internal to the sdk which says to skip the normalization for the properties in the array that are given to it. They don't necessarily have to be custom, it is whatever we provide it with.